### PR TITLE
Shield.io release badget

### PR DESCRIPTION
**Pull request by HawkonDK**
Added Shield.io release badget and a tab error in one of the code examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [jQuery Validation Plugin](https://jqueryvalidation.org/) - Form validation made easy
 ================================
 
+[![release](https://img.shields.io/github/release/jquery-validation/jquery-validation.svg)](https://github.com/jquery-validation/jquery-validation/releases/latest)
```
Comment (staabm) [inline README.md]: @Arkni wdyt...? I dont think we need this badge?

Comment (HawkonDK) [reply README.md]: fair enough just nice to see the version number so you don't need to go under releases to see if there are any new

Comment (Arkni) [reply README.md]: It's a good addition. I've seen a number of npm packages doing the same. It gives the visitor a glimpse of the current version without the need to check the changelog or the releases tab (as @HawkonDK said)

**Comment by staabm** [comment]
Thx

**Pull request merged**